### ISSUE TITLE
Bugs with TiConstruct integration.

### ISSUE THREAD

**Issue by OneEyeMaker** [ReikaKalseki/Reika_Mods_Issues#293]
Hello. I found some bugs in RotaryCraft. All of its related to TiCon integration.
1) Crash, when I try to view recipe of some tool parts or I try to view all recipes of Blast furnace.
Report: http://pastebin.com/UY5P00hp
That crash happens with and without ExtraTiC (and with ExtraTiC but with disabled RotaryCraft compatibility)
2) Some tools/armor are uncraftable at all. For example, I can't craft crossbow with bedrock crossbow limb.
3) Bedrock/Bedrock alloy can't be melted into liquid in smeltery/another machine => bolts are uncraftable (if specific TiConstruct config option is disabled; it's disabled by default).
.

**Comment by ReikaKalseki** [comment]
I have tested, and they work fine for me.

1) Contains none of my code.

2) Some - though not a crossbow - are intentionally uncraftable. What parts?

3) Bedrock alloy is not intended to be meltable. Bedrock TiC crafting is done in the Blast Furnace.

**Comment by OneEyeMaker** [comment]
1) I tested this bug only with TiCon (and core for it), RotaryCraft (and core for it) and NEI (+ core + addons).
Result: http://pastebin.com/3ETkXkJQ
Please, read lines from 46 to 51. TiCon tries to find ... fletching?!...

2) http://i.imgur.com/HpWj6bz.png

And I noticed, that parts (from Bedrock Alloy/HSLA Steel) haven't all required features.
Compare this:
http://i.imgur.com/4ElKqNn.png
and this:
http://i.imgur.com/r8SwZnY.png

**Comment by ReikaKalseki** [comment]
?

**Comment by OneEyeMaker** [comment]
1) As I understand, TiCon tries to find HSLA Steel fletching (Aux: 81 => it's ID for HSLA Steel tool parts defined in RotaryCraft.cfg by default). But Rotary doesn't contain this item (fletching from metal - it's strange...)
2) On screenshot: parts made from Bedrock Alooy haven't 'Draw speed' and 'Arrow speed' features. As I understand, these items haven't required tag in NBT => TiCon considers these parts as 'broken'.

**Comment by ReikaKalseki** [comment]
Both are related and come from how I create items for the TiC tools.

I have worked out that each tool part is its own item ID, where the material ID (an internal TiC ID) is the damage value of the tool part.
I was not aware any of the tool parts have specific NBT (only the tools themselves, with the "InfiTool" tag).

The bedrock side of it also happens because in v8, I used the TiC system of crafting, which no doubt automatically assigned the correct NBT tags. Now that it is done in the Blast Furnace, and I create the ItemStack myself, it probably is.

As for the fletching, I am unfamiliar with the weaponry side of TiC, and just added a bunch of automatic handling. It _should_ have created the part like anything else, even if it was not useful. Also, the crash is a crash from Mantle on index 3, not 81, so I am not convinced it is related to this.

Either way, feel free to ping whoever the best person on the TiC team is to ask.
